Code/BuildSystem/CMake/ezUtilsVars.cmake

```diff
@@ -59,4 +59,7 @@ mark_as_advanced(FORCE EZ_ENABLE_COMPILER_STATIC_ANALYSIS)
 ### Qt
 # set (EZ_VCPKG_INSTALL_QT OFF CACHE BOOL "Whether to install Qt via vcpkg.")
 
-
+######################################
+### SFML
```

Comment:
Shouldn't we remove all SFML references instead?

Reply:
I will remove all of SFML together with my glfw work.